test/api/qcFlags.test.js

```diff
@@ -217,6 +217,35 @@ module.exports = () => {
         });
     });
 
+    describe('GET /api/qcFlags/synchronous', () => {
+        it('should successfully fetch synchronous flags', async () => {
```

Comment:
You should put more specific names, here you have 2 tests with the same name

Reply:
done